docs/types/value-types.rst

```diff
@@ -337,6 +337,10 @@ on ``call``.
 
 .. _contract_types:
 
+* ``code`` and ``codehash``
+
+You can query the deployed code for any smart contract. Use ``code`` to get the EVM bytecode as a string, which might be empty. Use ``codehash`` get the Keccak-256 hash of that code.
+
```

Comment:
Looks good, but I guess the right place for it would be a few lines up - probably right before the "All contracts can be converted to ``address`` type..." note.

Reply:
Good note. Thank you, fixed at 5be61b2ea362b3d84ddd3892dc791fea28e4a768